tigerpath/majors_and_certificates/majors/ANT_2018.json

```diff
@@ -120,54 +151,137 @@
+                {
+                  "name": "Human Evolution",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "A course on Human Evolution or ANT 215 Human Adaptation or ANT 309 Forensic Anthropology or ANT 201 (beginning Fall 2020) or a cognate¹ biological course. Note: The department will accept Evolutionary Biology (EEB 309) and Life on Earth (EEB 211/MOL 211) as cognates if taken in AY 2019 to AY 2021 by students declaring the ANT concentration in Spring 2020 or earlier. These courses may be used to substitute for a required biological anthropology course offered by the Anthropology Department.",
+                  "course_list": [
+                    "ANT 215",
+                    "ANT 309",
+                    "ANT 201",
+                    "EEB 309",
+                    "EEB 211/MOL 211"
+                  ]
+                },
+                {
+                  "name": "Required Courses",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": null,
+                  "req_list": [
+                    {
+                      "name": null,
+                      "max_counted": 1,
+                      "min_needed": 1,
+                      "explanation": null,
+                      "course_list": [
+                        "ANT 300*"
+                      ]
+                    },
+                    {
+                      "name": null,
+                      "max_counted": 1,
+                      "min_needed": 1,
+                      "explanation": null,
+                      "course_list": [
+                        "ANT 301*"
+                      ]
+                    }
+                  ]
+                }
               ]
             },
             {
               "name": "Electives",
               "max_counted": 1,
               "min_needed": 5,
-              "explanation": "Examples of MedAnth electives are shown below. Elective courses are typically taught every other year, although some may be offered annually and others less frequently. A list of pre-approved MedAnth courses will be published each semester before course enrollment begins.",
+              "explanation": "Elective courses are typically taught every other year, although some may be offered annually and others less frequently. A list of pre-approved MedAnth courses will be published each semester before course enrollment begins.",
               "req_list": [
                 {
-                  "name": "Mind",
-                  "max_counted": 5,
-                  "min_needed": null,
-                  "explanation": "Courses pertaining to the mind: Psychological Anthropology (ANT 305); Mind, Body, and Bioethics in Japan and Beyond (EAS 312/ANT 312)",
-                  "course_list": [
-                    "ANT 305",
-                    "EAS 312/ANT 312"
+                  "name": "MedAnth Electives",

```

Comment:
It seems that the structure of the elective courses has changed too.
The department now requires two, one, and two courses, in separate categories. See [here](https://anthropology.princeton.edu/undergraduate/tracks/medical-anthropology).